en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -0,0 +1,17 @@
+# Silverstripe CMS 5.0.0-beta1
+
+## Post CMS5 beta freeze
+One of the primary purposes of the beta testing is to give third party maintainers developers an opportunity to upgrade their modules to be compatible with CMS5. Once the CMS5 beta is tagged, our aim is to:
+Make no further breaking API changes will be made to the CMS5 codebase.
+Have all supported dependencies upgraded to a supported major.
```

Comment:
```suggestion
- make no further breaking API changes to the CMS5 codebase
- have all supported dependencies upgraded to a supported major version.
```
Conforming with the writing standards on confluence, plus some tweaks for readability